hypertrace-graphql-span-processing-schema/src/main/java/org/hypertrace/graphql/spanprocessing/schema/rule/ExcludeSpanRule.java

```diff
@@ -1,12 +1,15 @@
 package org.hypertrace.graphql.spanprocessing.schema.rule;
 
+import graphql.annotations.annotationTypes.GraphQLDescription;
 import graphql.annotations.annotationTypes.GraphQLField;
 import graphql.annotations.annotationTypes.GraphQLName;
 import graphql.annotations.annotationTypes.GraphQLNonNull;
 import java.time.Instant;
 import org.hypertrace.core.graphql.common.schema.id.Identifiable;
 
 @GraphQLName(ExcludeSpanRule.TYPE_NAME)
+@GraphQLDescription(
+    "Exclude span rules are the set of rules created using filters on span attributes based on which we exclude spans at source from reaching the platform")
```

Comment:
This is good - some minor tweaks to make it a little more abstract (the where is an impl detail):

"An ExcludeSpanRule describes a set of conditions based on span attributes. When matched, a span is discarded and not eligible for further processing."